ds_judgements_public_ui/templates/includes/results_filters_inputs.html

```diff
@@ -17,43 +16,32 @@
         <p class="structured-search__from-date-error-message"
            id="from-date-error">{{ context.errors|errors_for_field:"from_date" }}</p>
       {% endif %}
-      {% if text_date_input %}
-        <input type="hidden" name="text_date_input" value="True" />
```

Comment:
not quite sure what the point of this was before anyway? For my understanding does anyone know if this was serving a particular purpose?